[RCCA-23805] Cannot remove `ruleSet` via terraform resource `confluent_schema` (#654)
confluentinc/terraform-provider-confluent: docs/resources/confluent_schema.md

-> **Note:** The Confluent Cloud Console uses the following default values: `on_success = "NONE"` and `on_failure = "ERROR"`. However, the TF Provider sets its defaults to `on_success = "NONE,NONE"` and `on_failure = "ERROR,ERROR"`.

> -> **Note:** To delete all schema rules from a schema, remove the ruleset block completely from the terraform file.

Reviewer: ```suggestion
-> **Note:** To delete all schema rules from a schema, remove the ruleset block completely from the Terraform configuration file.
```

> -> **Note:** To delete all schema rules from a schema, remove the ruleset block completely from the Terraform configuration file..

Reviewer 2: ```suggestion
-> **Note:** To delete all schema rules from a schema, remove the ruleset block completely from the Terraform configuration file.
```

-> **Note:** The Confluent Cloud Console uses the following default values: `on_success = "NONE"` and `on_failure = "ERROR"`. However, the TF Provider sets its defaults to `on_success = "NONE,NONE"` and `on_failure = "ERROR,ERROR"`.

> -> **Note:** To delete all schema rules from a schema, remove the ruleset block completely from the Terraform configuration file.

Reviewer: ![image](https://github.com/user-attachments/assets/eb9f88d6-c91f-438a-b62d-1944e88f5ced)
```
  ruleset {
    domain_rules {
      name = "encryptPII"
```

```suggestion
-> **Note:** To delete all domain rules from an existing schema, remove the `ruleset` block from the `confluent_schema` resource in the Terraform configuration.
```
